Src/WitsmlExplorer.Frontend/components/Modals/CopyMnemonicsModal.tsx

```diff
@@ -0,0 +1,231 @@
+      const deleteJob: DeleteComponentsJob = {
+        toDelete: createComponentReferences(
+          targetComponentsToDelete.map((component) => component.mnemonic),
+          targetParent,
+          ComponentType.Mnemonic,
+          targetServer.url
+        )
+      };
+
+      const copyJob: CopyComponentsJob = {
+        source: sourceReferences,
+        target: targetReference
+      };
+
+      const replaceJob: ReplaceComponentsJob = { deleteJob, copyJob };
+
+      if (sourceServer) {
+        await JobService.orderJobAtServer(
+          JobType.ReplaceComponents,
+          replaceJob,
+          targetServer,
+          sourceServer
+        );
+      } else {
+        await JobService.orderJob(JobType.ReplaceComponents, replaceJob);
+      }
+    }
+  }
+
+  return (
+    <>
+      <ModalDialog
+        heading={`Paste`}
+        confirmText={`Submit`}
+        content={
+          <ContentLayout>
+            <TextLayout>Choose paste option:</TextLayout>
+            <RadioLayout>
+              <RadioItemLayout>
+                <div>
+                  <Radio
+                    checked={
+                      selectedCopyMnemonicsType ===
+                      CopyMnemonicsType.DeleteInsert
+                    }
+                    onChange={() =>
+                      setCopyMnemonicsType(CopyMnemonicsType.DeleteInsert)
+                    }
+                  />
+                </div>
+                <div>
+                  <div>Delete/Insert</div>
+                  <div>
+                    Delete target mnemonics before copying. The mnemonics will
+                    become equal on the source and target server afterwards.
+                  </div>
+                </div>
+              </RadioItemLayout>
+              <RadioItemLayout>
+                <div>
+                  <Radio
+                    checked={
+                      selectedCopyMnemonicsType === CopyMnemonicsType.Paste
+                    }
+                    onChange={() =>
+                      setCopyMnemonicsType(CopyMnemonicsType.Paste)
+                    }
+                  />
+                </div>
+                <div>
+                  <div>Paste</div>
+                  <div>
+                    Data on target server will be overwritten by data on the
+                    source. Data on target server outside of the
+                    startIndex/endIndex will be kept as is.
+                  </div>
+                </div>
+              </RadioItemLayout>
+            </RadioLayout>
+          </ContentLayout>
+        }
+        onSubmit={() => onSubmit()}
+        isLoading={isLoading}
+      />
+    </>
+  );
+};
+
+const ContentLayout = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 0.25rem;
+`;
+
+const TextLayout = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 0.25rem;
+`;
+
+const RadioLayout = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 0.25rem;
```

Comment:
Would be nice to have a bit more distance between the two options (and maybe also for the ContentLayout)